package.json

```diff
@@ -68,7 +68,8 @@
 		"@wordpress/scripts": "^24.6.0",
 		"@wordpress/url": "^3.23.0",
 		"concurrently": "^7.6.0",
-		"eslint-plugin-jest": "^27.1.6",
+		"dayjs": "^1.11.6",
+		"eslint-plugin-jest": "^27.1.5",
```

Comment:
Any reason this constraint has dropped down a patch version?

Reply:
My bad .... will revert it